code/datums/components/wet.dm

```diff
@@ -0,0 +1,68 @@
+/datum/component/wet
```

Comment:
so wetness availability is determined by a generic item flag and you need to use GetComponent to check wetness. I feel like you would be better off making this a datum since its not self contained

Reply:
i was going to make a datum at first, but i thought components would be more optimal incase it needed to be expanded in the future